bitnami/mongodb/values.yaml

```
@@ -133,6 +133,9 @@ auth:
   ## ref: https://docs.mongodb.com/manual/tutorial/enable-authentication/
   ##
   enabled: true
+  ## @param auth.rootPassword MongoDB&reg; root user
+  ##
+  rootUser: "root"
```

Double-quotes aren't necessary.
```suggestion
  rootUser: root
```